starter-code/index.html

```diff
@@ -0,0 +1,114 @@
+        <img class="d-block w-100" src="./img/slider-1.jpg" alt="First slide">
+        <div class="overlay"></div>
+        <div class="carousel-caption d-none d-md-block">
+          <h5>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</h5>
+          <p>ODuis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="./img/slider-2.jpg" alt="Second slide">
+        <div class="overlay"></div>
+        <div class="carousel-caption d-none d-md-block">
+          <h5>FUSCE SCELERISQUE NIBH AC NISL LUCTUS INTERDUM.</h5>
+          <p>Nam luctus velit ultricies vestibulum eleifend. Proin imperdiet varius rutrum. Curabitur placerat lacus eu ligula pellentesque euismod? Aliquam volutpat augue id nibh auctor, at varius dui semper? Suspendisse non pellentesque sapien, et sagittis erat. Quisque et dignissim elit. Sed sit amet sapien ac quam vestibulum suscipit ac id diam. Morbi accumsan est vitae tellus vulputate […]</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="./img/slider-3.jpg" alt="Third slide">
+        <div class="overlay"></div>
+        <div class="carousel-caption d-none d-md-block">
+          <h5>FALAURIS SOLLICITUDIN QUIS DOLOR VENENATIS FACILISIS</h5>
+          <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="./img/slider-4.jpg" alt="Fourth slide">
+        <div class="overlay"></div>
+        <div class="carousel-caption d-none d-md-block">
+          <h5>SUSPENDISSE ORNARE, FELIS FEUGIAT SUSCIPIT PHARETRA</h5>
+          <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+        </div>
+      </div>
+    </div>
+    <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```

Comment:
Arrows are not working for me. Bootstrap is also using jQuery, so you may have something broken related to this jquery.